paper/ms.tex

```diff
@@ -928,9 +926,24 @@ \subsection{Binarity}
 We incorporated each of these sources of information into a column in
 Table~\ref{tab:thetable} that expresses whether any given source
 passed each of the binarity indicators.
-We discuss some important--and complicated--aspects of the systems
+We discuss some important---and complicated---aspects of the systems
 that seem to be binary in Section~\ref{subsec:discbinary}.
 
+\subsection{Constraints on the presence of a planet}
+\label{subsec:planet_constraints}
+We searched for evidence of a transiting exoplanet for 10 of the brightest
```

Comment:
This was the selection function.  However I would rather leave this out.  The full exercise here is injection-recovery over a grid of Rp/P tuples, and pretty expensive to demonstrate.  (And would probably require adding figures to give a flavor for whether we should really be surprised at not having seen any yet!)